src/System.Private.CoreLib/shared/System/Text/Rune.cs

```diff
@@ -987,6 +1012,15 @@ public bool TryEncodeToUtf8Bytes(Span<byte> destination, out int bytesWritten)
             return false;
         }
 
+        public bool TryEncodeToUtf8Bytes(Span<byte> destination, out int bytesWritten)
+        {
+            // This method was renamed to TryEncodeAsUtf8. We'll leave this copy of
```

Comment:
Nit: could you add "TODO" to the text, and the associated issue number?  Just in case we forget to remove this in the near future, it'll make it easier to spot that it should be removed later.